hello-world/app/js-vanilla/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "templates-app-vanilla-javascript",
+  "description": "Polywrap dApp Vanilla Javascript Template",
+  "private": true,
+  "version": "0.0.1-prealpha.1",
+  "scripts": {
+    "start": "serve public",
+    "build": "yarn codegen && rollup -c",
+    "codegen": "npx w3 app codegen"
+  },
+  "dependencies": {
+    "@web3api/client-js": "latest",
+    "serve": "^13.0.2",
+    "toastify-js": "^1.11.2"
```

Comment:
Please remove the `^` from all node module versions, we should be fixing our dependencies as much as possible.